oracle/price-feeder/oracle/oracle.go

```diff
@@ -548,7 +577,11 @@ func (o *Oracle) tick(
 		return err
 	}
 
-	exchangeRatesStr := GenerateExchangeRatesString(o.GetPrices())
+	prices := o.GetPrices()
+
+	// filter for whitelisted denominations so that extra oracle prices are not penalized
+	filteredPrices := filterPricesByDenomList(prices, oracleParams.Whitelist)
```

Comment:
Where is this whitelist coming from?

Reply:
It's part of the oracle params structure:
```
$ seid query oracle params
params:
  lookback_duration: "3600"
  min_valid_per_window: "0.050000000000000000"
  reward_band: "0.020000000000000000"
  slash_fraction: "0.000000000000000000"
  slash_window: "216000"
  vote_period: "2"
  vote_threshold: "0.667000000000000000"
  whitelist:
  - name: ueth
  - name: ubtc
  - name: uusdc
  - name: uusdt
```